app/namespaces/data_provider/data_provider_schema.py

```diff
@@ -34,6 +35,14 @@ class RecordsSchema(Schema):
     start_date = fields.String()
     end_date = fields.String()
 
+class PaginatedRecordsSchema(RecordsSchema):
```

Comment:
add the param columns to this schema check since it's part of the payload

Reply:
Just want to make sure, since `PaginatedRecordsSchema` inherits from `RecordsSchema` - shouldn't the `columns` param already be included?

Reply:
good point! no change needed here then.